client/src/components/PortfolioFilters.tsx

```diff
@@ -498,55 +498,59 @@ function MinMaxSelect(props: {
 }) {
   const { options, onApply, id, onFocusInput } = props;
   const [minMax, setMinMax] = React.useState<FilterNumberRange>(MINMAX_DEFAULT);
-  const [minMaxErrors, setMinMaxErrors] = React.useState([false, false]);
+  const [minMaxErrors, setMinMaxErrors] = React.useState<MinMaxErrors>([false, false, undefined]);
```

Comment:
out of curiosity, why did you define `FilterNumberRange` type as an [array](https://github.com/JustFixNYC/who-owns-what/blob/6d7ecbd6295c7f86be2814228fb4bcb18304e5ea/client/src/components/PropertiesList.tsx#L16)? I feel like it's a little unintuitive for me personally to see values accessed with indices, when we could've done something like `FilterNumberRange.min`, `FilterNumberRange.max`, and `FilterNumberRange.error`

Reply:
Originally it was because react-table had a default option for filtering by a range and they use this format. But yeah now that there are all these uses of that data before passing it to the table filter function I should probably change it to what you're suggesting here and just change it to that array only when the filtering happens. I can make a new PR that just makes this change

Reply:
that would be awesome!! approved the PR regardless bc that ^ seemed like a preference and best practice more than a necessity atm